development/contracts/ForkableBridge.sol

```diff
@@ -0,0 +1,323 @@
+pragma solidity ^0.8.17;
+
+import {PolygonZkEVMBridge, IBasePolygonZkEVMGlobalExitRoot} from "@RealityETH/zkevm-contracts/contracts/inheritedMainContracts/PolygonZkEVMBridge.sol";
+import {IPolygonZkEVMBridge} from "@RealityETH/zkevm-contracts/contracts/interfaces/IPolygonZkEVMBridge.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20Metadata} from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import {TokenWrapped} from "@RealityETH/zkevm-contracts/contracts/lib/TokenWrapped.sol";
+import {IForkableBridge} from "./interfaces/IForkableBridge.sol";
+import {IForkonomicToken} from "./interfaces/IForkonomicToken.sol";
+import {ForkableUUPS} from "./mixin/ForkableUUPS.sol";
+
+contract ForkableBridge is IForkableBridge, ForkableUUPS, PolygonZkEVMBridge {
+    bytes32 public constant HARD_ASSET_MANAGER_ROLE =
+        keccak256("HARD_ASSET_MANAGER_ROLE");
+
+    // @inheritdoc IForkableBridge
+    function initialize(
+        address _forkmanager,
+        address _parentContract,
+        uint32 _networkID,
+        IBasePolygonZkEVMGlobalExitRoot _globalExitRootManager,
+        address _polygonZkEVMaddress,
+        address _gasTokenAddress,
+        bool _isDeployedOnL2,
+        address hardAssetManger,
+        uint32 lastUpdatedDepositCount,
+        bytes32[_DEPOSIT_CONTRACT_TREE_DEPTH] calldata depositTree
+    ) public virtual initializer {
+        ForkableUUPS.initialize(_forkmanager, _parentContract, msg.sender);
+        PolygonZkEVMBridge.initialize(
+            _networkID,
+            _globalExitRootManager,
+            _polygonZkEVMaddress,
+            _gasTokenAddress,
+            _isDeployedOnL2,
+            lastUpdatedDepositCount,
+            depositTree
+        );
+        _setupRole(HARD_ASSET_MANAGER_ROLE, hardAssetManger);
+    }
+
+    /**
+     * @notice Function to send hard assets to the children-bridge contracts
+     * @param token Address of the token
+     * @param amount Amount of tokens to transfer
+     * @param to Address to transfer the tokens to
+     */
+    function manageHardAssets(
```

Comment:
"manage" isn't very descriptive, can we express what this is for? is it something like transferHardAssetsToChild?